### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by EthanHicks1**
Your bad.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,4 +2,4 @@
 My code is perfect... don't you dare try to review it
 
 Every one knows coders don't make mistakes.
-My programs always for on the firs try.
```
Comment (EthanHicks1) [inline README.md:5]: You had a typo.

File: README.md [hunk 2]
```diff
@@ -2,4 +2,4 @@
...
+My programs always on the first try.
```
Comment (robobenklein) [inline README.md:5]: Hmmmmm

Comment (EthanHicks1) [inline README.md:5]: this is a little better

File: README.md [hunk 3]
```diff
@@ -2,4 +2,4 @@
 My code is perfect... don't you dare try to review it
```
Comment (robobenklein) [inline README.md:2]: Don't worry I wont.

**Review by robobenklein — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (davidclevenger) [reply README.md:5]: Looks good

**Pull request merged**